src/nrniv/CMakeLists.txt

```diff
@@ -26,6 +26,11 @@ set_property(
   PROPERTY COMPILE_DEFINITIONS NRNUNIT=1)
 
 add_executable(modlunit ${NRN_MODLUNIT_SRC_FILES})
+
+if (${NRN_FRONTEND_CXX_FLAGS})
+    target_compile_options(modlunit ${NRN_FRONTEND_CXX_FLAGS})
+endif()
```

Comment:
@alkino : see line number 41 :
* I think if you have multiple flags we have to use `separate_arguments` ?
* If you set target_compile_options, other options set via CMAKE_CXX_FLAGS are removed? I think I saw these new flags just being appended.
* Did you to test this on BB5 with the example I posted in the issue?
* Note that we are adding `CMAKE_C_FLAGS_DEBUG` on line 41 and it needs to be removed

Reply:
Yes it's appended, you talk in the chat about the fact we can add new flags to remove action of flags added inside CMAKE_CXX_FLAGS
it's what you asked

Reply:
why should we remove `${CMAKE_C_FLAGS_DEBUG}` ?